src/Sarif.Multitool.Library/Rules/RuleResources.Designer.cs

```diff
@@ -822,9 +822,9 @@ internal class RuleResources {
         /// <summary>
         ///   Looks up a localized string similar to Rule metadata should provide information that makes it easy to understand and fix the problem.
         ///
-        ///Provide the &apos;name&apos; property, which contains a &quot;friendly name&quot; that helps users see at a glance the purpose of the rule. For uniformity of experience across all tools that produce SARIF, the friendly name should be a single Pascal identifier, for example, &apos;ProvideRuleFriendlyName&apos;.
+        ///Provide the &apos;name&apos; property, which contains a &quot;friendly name&quot; that helps users see at a glance the purpose of the rule. For uniformity of experience across all tools that produce SARIF, the friendly name should be a single Pascal-case identifier, for example, &apos;ProvideRuleFriendlyName&apos;.
         ///
-        ///Provide the &apos;helpUri&apos; property, which contains a URI where users can find detailed information about the rule. This information s [rest of string was truncated]&quot;;.
+        ///Provide the &apos;helpUri&apos; property, which contains a URI where users can find detailed information about the rule. This informat [rest of string was truncated]&quot;;.
```

Comment:
>informat [](start = 137, length = 8)

Looks like you tried to remove that erroneous 's' but maybe deleted too much of the string?

Reply:
This text is automatic generated when we change the resource itself. so, its breaking in a unusual space.

---
In reply to: [533635356](https://github.com/microsoft/sarif-sdk/pull/2161#discussion_r533635356) [](ancestors = 533635356)